### DISCUSSION TITLE
Crun2 error after mastercontainer update

### DISCUSSION THREAD

**Discussion by loeffelpan** [nextcloud/all-in-one#4204] [Questions] [answered]
This was was discussed in the [podman thread](https://github.com/nextcloud/all-in-one/discussions/3487#discussioncomment-8173272) but without any result. So I would like to ask this directly in a dedicated thread.

As mentioned I'm using podman and everything works fine. But on any update of the mastercontainer itself I get stuck. The new mastercontainer gets created but cannot be started.
It throws `crun: cannot set memory swappiness with cgroupv2`.

I found an (unsolved) [issue](https://github.com/containers/podman/issues/13916) about cloning, what AIO doesn't do, but maybe this is related.
Any thought why this would happen?

**Comment by szaimen** [comment, marked as answer]
Hi, since [podman is not supported](https://github.com/nextcloud/all-in-one?tab=readme-ov-file#can-i-run-this-with-podman-instead-of-docker), please keep the discussion in this thread: https://github.com/nextcloud/all-in-one/discussions/3487
Thanks!

**Reply by loeffelpan** [reply]
I know that this isn't supported and supporting podman is not what I wanted. 
Let me clarify my questions:
Can you explain what you exactly do on mastercontainer update?
Is there anything with memory/swapiness tuning when creating a new container?
What is different to other container updates (which work)?

Please help me invetigate this further by myself. Then I can maybe open an issue to the podman devs.

**Reply by szaimen** [reply]
On mastercontainer update we simply start watchtower in a separate container that updates the mastercontainer.

**Reply by loeffelpan** [reply]
Thank you. That helped.
I found that error an watchtower issue.
See https://github.com/containrrr/watchtower/issues/1060.
